src/v2/Apps/Fair/Components/FairCollections/FairCollection.tsx

```diff
@@ -75,6 +75,8 @@ export const FairCollection: React.FC<FairCollectionProps> = ({
     }
   })
 
+  const count = collection.artworks.counts.total
```

Comment:
Will these fields always exist? I get a little nervous when I see a long chain with no `?` 😅

Reply:
It should exist! The `artworks` and `counts` object should exist always (and we do reference those directly in various filter components).